### ISSUE TITLE
Is anyone still developing this project? I find it very useful

### ISSUE THREAD

**Issue by falematte** [louietan/anki-editor#105]

**Comment by renatgalimov** [comment]
This particular branch seems to be dead. 

I use this fork:

https://github.com/orgtre/anki-editor

**Comment by orgtre** [comment]
Yes, indeed, the fork at https://github.com/orgtre/anki-editor is actively developed and I encourage contributions there. Also see [this issue](https://github.com/louietan/anki-editor/issues/101).
